network/meson.build

```diff
@@ -0,0 +1,12 @@
+# -*- mode: meson -*-
+
+if conf.get('ENABLE_NETWORKD', 0) == 1
+  install_data('80-container-host0.network',
+               '80-container-ve.network',
+               '80-container-vz.network',
+               '99-default.link',
```

Comment:
Those files are installed unconditionally under autotools. I would at least install 99-default.link always, as it's not strictly networkd specific, but used by udev. Making the installation of the .network files conditional seems reasonable.